recipes/isbnlib/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "isbnlib" %}
+{% set version = "3.10.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/isbnlib-{{ version }}.tar.gz
+  sha256: c9e6c1dcaa9dff195429373cf2beb3117f30b3fca43d7db5aec5a2d1f6f59784
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+
+test:
+  imports:
+    - isbnlib
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/xlcnd/isbnlib
+  summary: Extract, clean, transform, hyphenate and metadata for ISBNs (International Standard Book Number).
+  license: LGPL-3.0-or-later
+  license_file: LICENSE-LGPL-3.0.txt
```

```suggestion
  license_file: 
    - LICENSE-LGPL-3.0.txt
    - COPYRIGHT.txt
```
This needs to be included as well.